db/1.14/modules/ROOT/pages/database-stored-procedure.adoc

```diff
@@ -107,6 +107,7 @@ In the *Input parameters* field, specify values that create a map in which keys
 
 * Define each parameter name with single quotes for example, `'paramName': value`.
 * For default values either use DataWeave or a variable, so if you pass null, or an empty string the store procedure's default value won't be pushed by Mule runtime engine.
+* Enter input values with parameters only, do not hardcode the values within the *Stored Procedure* operation.
```

Comment:
I suggest making this two sentences:
Enter input values with parameters only. Do not hardcode the values within the *Stored Procedure* operation 
OR
add "and":
Enter input values with parameters only and do not hardcode the values within the *Stored Procedure* operation